app/models.py

```diff
@@ -141,7 +143,8 @@ def serialize(self):
             "organisations": [x.id for x in self.organisations if x.active],
             "password_changed_at": self.password_changed_at.strftime(DATETIME_FORMAT_NO_TIMEZONE),
             "permissions": self.get_permissions(),
-            "platform_admin": self.platform_admin,
+            "platform_admin_capable": self.platform_admin_capable,
```

Comment:
I do wonder whether we should have `"platform_admin": self.platform_admin_capable` here as a temporary measure. As it currently stands this is a breaking change which the older probably admin won't tolerate if we have a brief period of version skew while deploying or rolling back.

Reply:
Probably sensible if we wnated to avoid full downtime.

Reply:
Done